SayTheirNames/Source/Dependency/Networking/NetworkRequestor.swift

```diff
@@ -27,32 +27,33 @@ import Alamofire
 
 // MARK: - NetworkRequestor
 
-final class NetworkRequestor {
-    let concurrentQueue = DispatchQueue(label: "NetworkRequestor", attributes: .concurrent)
+final class NetworkRequestor: Dependency {
     let session: Session
-
+    let concurrentQueue = DispatchQueue(label: "NetworkRequestor", attributes: .concurrent)
+    private let headers = HTTPHeaders.init(["x-dates-epoch": "true"])
```

Comment:
could you make a separate PR for the date stuff, for visibility? 🙏

Reply:
Yup; gonna remove all of this logic into another PR for us.

Reply:
All moved.